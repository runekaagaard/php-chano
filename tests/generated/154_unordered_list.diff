001- <li>item 1</li>
001+ Fatal error: Call to undefined method DtlIter::unordered_list() in /home/rune/workspace/php-misc/php-dtl-iterator/tests/django-generated/154_unordered_list.php on line 7
002- 	<li>item 2</li>