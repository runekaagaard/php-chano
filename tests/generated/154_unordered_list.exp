<li>item 1</li>
	<li>item 2</li>