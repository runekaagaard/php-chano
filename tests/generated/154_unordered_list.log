
---- EXPECTED OUTPUT
<li>item 1</li>
	<li>item 2</li>
---- ACTUAL OUTPUT
Fatal error: Call to undefined method DtlIter::unordered_list() in /home/rune/workspace/php-misc/php-dtl-iterator/tests/generated/154_unordered_list.php on line 7
---- FAILED
