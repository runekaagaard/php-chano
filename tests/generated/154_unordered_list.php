<?php
include dirname(__FILE__) . '/../../DtlIter.php';
$items = new DtlIter(array(array('input' => array (
  0 => 'item 1',
  1 => 'item 2',
))));
foreach ($items as $i) echo $i->input->unordered_list();
