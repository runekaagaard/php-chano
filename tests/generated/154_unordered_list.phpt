--TEST--
A generated testfile for the "unorderedlist" filter.
--FILE--
<?php
include dirname(__FILE__) . '/../../DtlIter.php';
$items = new DtlIter(array(array('input' => array (
  0 => 'item 1',
  1 => 'item 2',
))));
foreach ($items as $i) echo $i->input->unorderedlist();
--EXPECT--
	<li>item 1</li>
	<li>item 2</li>