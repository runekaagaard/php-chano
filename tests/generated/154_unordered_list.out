Fatal error: Call to undefined method DtlIter::unordered_list() in /home/rune/workspace/php-misc/php-dtl-iterator/tests/django-generated/154_unordered_list.php on line 7